mapswipe_workers/mapswipe_workers/generate_stats/overall_stats.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+from mapswipe_workers import auth
+from mapswipe_workers.definitions import logger
+from mapswipe_workers.utils import geojson_functions
+
+
+def get_project_static_info(filename):
+
+    pg_db = auth.postgresDB()
+    sql_query = """
+        COPY (
+            SELECT 
+                project_id
+                ,name
+                ,project_details
+                ,look_for
+                ,project_type
+                ,status
+                ,ST_Area(geom::geography)/1000000 as area_sqkm
+                ,ST_AsText(geom) as geom
+                ,ST_AsText(ST_Centroid(geom)) as centroid
+            FROM projects
+        ) TO STDOUT WITH CSV HEADER"""
+
+    with open(filename, 'w') as f:
+        pg_db.copy_expert(sql_query, f)
+
+    del pg_db
+    logger.info('got projects from postgres.')
+
+    df = pd.read_csv(filename)
```

Comment:
Could it be more efficient to read the dataframe directly from the DB with [pd.read_sql_query](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_sql_query.html#pandas.read_sql_query) ?